templates/base/scripts/liferay_entrypoint.sh

```diff
@@ -42,6 +42,8 @@ function start_liferay {
 
 	START_LIFERAY_PID=$!
 
+	echo "${START_LIFERAY_PID}" >> /opt/liferay/liferay.pid
```

Comment:
Nice! But should this go in a more standard location like `/var/run/liferay.pid`